companies/two.json

```diff
@@ -0,0 +1,19 @@
+{
+    "slug": "two",
```

Comment:
I agree that `two` is a little too generic. Let's use `technische-werke-osning` instead.